test/private/initializeGlobalState.m

```diff
@@ -0,0 +1,47 @@
+function orig = initializeGlobalState()
+% Initialize global state. Set various properties of the graphical root to ensure reliable output of the ACID testsuite.
+% See #542 and #552
+    fprintf('Initialize global state...\n');
+
+    %--- Define desired global state properties
+    % See http://undocumentedmatlab.com/blog/getundoc-get-undocumented-object-properties
+    new.defaultAxesColorOrder.val   =               ...
```

Comment:
Ok! Let me see, whether I can tackle this now or is there something else I should focus one, while I have some spare time?